tutorials/using-cloud-vpn-with-cisco-asr/index.md

```diff
@@ -0,0 +1,893 @@
+  Package License (SLASR1-AIS)
+* IPsec RTU license (FLASR1-IPsec-RTU)
+* Encryption HW module (ASR1002HX-IPsecHW(=) and ASR1001HX-IPsecW(=)) and Tiered
+  Crypto throughput license which applies to ASR1002-HX and ASR1001-HX chassis
+  only.
+
+For a detailed ASR 1000 Series Router license information, refer to the
+[ASR 1000 Routers Ordering Guide](http://www.cisco.com/c/en/us/products/collateral/routers/asr-1000-series-aggregation-services-routers/guide-c07-731639.html).
+
+### IPsec parameters
+
+For the Cisco ASR 1000 IPsec configuration, the following details will be used:
+
+|Parameter | Value|
+--------- |  -----
+|IPsec Mode | `Tunnel mode` |
+|Auth protocol | `Pre-shared-key` |
+|Key Exchange | `IKEv2` |
+|Start | `Auto` |
+|Perfect Forward Secrecy (PFS) | `Group 16` |
+|Dead Peer Detection (DPD) | `60 5 periodic` |
+
+The IPsec configuration used in this guide is specified below:
+
+| Cipher Role | Cipher |
+| ------------| -------|
+| Encryption | `esp-aes 256 esp-sha-hmac` |
+| Integrity | `sha256` |
+| Diffie-Hellman (DH) | `group 16` |
+| Lifetime | `36,000 seconds (10 hours)` |
+
+## Configuration – GCP
+
+### IPsec VPN using static routing
+
+This section provides the steps to create Cloud VPN on GCP. For a basic overview
+of what is Cloud VPN take a look at [this document][compute_vpn]. There are two
+ways to create VPN on GCP, using Google Cloud Platform Console and the `gcloud`
+command-line tool. The upcoming section provide details to both in detail below:
+
+[compute_vpn]: https://cloud.google.com/compute/docs/vpn/overview
+
+#### Using the Google Cloud Platform Console
+
+1.  [Go to the VPN page](https://console.cloud.google.com/networking/vpn/list)
+    in the Google Cloud Platform Console.
+1.  Click **Create VPN connection**.
+1.  Populate the following fields for the gateway:
+
+    * **Name** — The name of the VPN gateway. This name is displayed in the
+      console and used by the `gcloud` command-line tool to reference the gateway.
+    * **VPC network** — The VPC network containing the instances the VPN gateway
+      will serve. In this case it is `vpn-scale-test-cisco`, a
+      [custom VPC network](https://cloud.google.com/compute/docs/vpc/using-vpc#create-custom-network).
+      Ensure this network does not conflict with your on-premises networks.
+    * **Region** — The region where you want to locate the VPN gateway.
+      Normally, this is the region that contains the instances you wish to
+      reach. Example: `us-east1`
+    * **IP address** — Select a pre-existing [static external IP address](https://cloud.google.com/compute/docs/ip-addresses#reservedaddress).
+      If you don't have a static external IP address, you can create one by
+      clicking **New static IP address** in the pull-down menu. Selected
+      `vpn-scale-test0` for this guide.
+1.  Populate fields for at least one tunnel:
+
+    * **Peer IP address** — Enter your peer public IP address here, with the
```

Comment:
in the doc, we use "on-premises" to clarify which peer (both sides of the tunnel are peers). not sure if you want to do the same for this entire article.